test/integration/targets/nxos_tms_destgroup/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+# Uncomment in the future if needed.
+#  - prepare_nxos_tests
```

Comment:
Why skip `prepare_nxos_tests` ?

Reply:
We don't explicitly need it right now and it just adds runtime.

Reply:
👍